### PR TITLE
BUG make sure to patch libgfortran correctly

### LINKED ISSUE #10: libgfortran patches not applied correctly to numpy on osx

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #10

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@scopatz @jjhelmus @isuruf for viz and code review

**Comment by beckermr** [comment]
Ack hold off. This still did not fix it.

**Comment by beckermr** [comment]
Mmmmk. I have fixed this. Note not all versions of numpy list a libgfortran dep. Is this a mistake?

```
numpy-1.9.3-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.11.2-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.10.4-py36_blas_openblas_205.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_205
numpy-1.13.1-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.14.5-py27_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=2.7,<2.8.0a0'] None py27_blas_openblash24bf2e0_200
numpy-1.10.4-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.13.1-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.13.3-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.11.0-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.14.2-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.10.4-py34_blas_openblas_204.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.4*'] None py34_blas_openblas_204
numpy-1.8.2-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.8.2-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_201
numpy-1.9.3-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.11.3-py36_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_203
numpy-1.13.1-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.9.3-py34_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_202
numpy-1.11.2-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.14.5-py36_blas_openblashd3ea46f_200.tar.bz2 ['blas 1.1 openblas', 'libgfortran-ng >=3.0', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=3.6,<3.7.0a0'] None py36_blas_openblashd3ea46f_200
numpy-1.11.1-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.14.5-py35_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=3.5,<3.6.0a0'] None py35_blas_openblash24bf2e0_200
numpy-1.13.1-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_201
numpy-1.14.0-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.11.2-py34_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_202
numpy-1.10.4-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.13.0-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.11.3-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.6*'] None py36_blas_openblas_201
numpy-1.9.3-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.11.3-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.8.2-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.14.0-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.11.2-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.11.3-py35_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_203
numpy-1.10.4-py35_blas_openblas_204.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.5*'] None py35_blas_openblas_204
numpy-1.10.4-py27_blas_openblas_205.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_205
numpy-1.12.1-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.10.4-py35_blas_openblas_205.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_205
numpy-1.11.0-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.11.0-py27_blas_openblas_100.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2*', 'python 2.7*'] None py27_blas_openblas_100
numpy-1.14.4-py35_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=3.5,<3.6.0a0'] None py35_blas_openblash24bf2e0_200
numpy-1.9.3-py36_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_203
numpy-1.9.3-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.0-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.12.1-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.11.2-py35_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_202
numpy-1.11.0-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.10.4-py27_blas_openblas_204.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 2.7*'] None py27_blas_openblas_204
numpy-1.10.4-py27_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_202
numpy-1.11.3-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.14.3-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=3.5,<3.6.0a0'] None py35_blas_openblas_200
numpy-1.11.3-py34_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_202
numpy-1.14.5-py27_blas_openblashd3ea46f_200.tar.bz2 ['blas 1.1 openblas', 'libgfortran-ng >=3.0', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=2.7,<2.8.0a0'] None py27_blas_openblashd3ea46f_200
numpy-1.11.3-py27_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_203
numpy-1.12.1-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.11.3-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.13.1-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.9.3-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.14.3-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=2.7,<2.8.0a0'] None py27_blas_openblas_200
numpy-1.10.4-py34_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_202
numpy-1.9.3-py27_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_202
numpy-1.13.1-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.14.4-py36_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=3.6,<3.7.0a0'] None py36_blas_openblash24bf2e0_200
numpy-1.11.3-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.8.2-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.9.3-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.14.4-py27_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=2.7,<2.8.0a0'] None py27_blas_openblash24bf2e0_200
numpy-1.11.0-py34_blas_openblas_100.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2*', 'python 3.4*'] None py34_blas_openblas_100
numpy-1.11.3-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.8.2-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.13.0-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.11.1-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.10.4-py35_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.5*'] None py35_blas_openblas_203
numpy-1.11.2-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.14.2-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.11.0-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.11.2-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.3-py34_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_203
numpy-1.14.5-py36_blas_openblash24bf2e0_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=3.6,<3.7.0a0'] None py36_blas_openblash24bf2e0_200
numpy-1.11.3-py35_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_202
numpy-1.14.5-py35_blas_openblashd3ea46f_200.tar.bz2 ['blas 1.1 openblas', 'libgfortran-ng >=3.0', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=3.5,<3.6.0a0'] None py35_blas_openblashd3ea46f_200
numpy-1.10.4-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.13.2-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.12.0-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.12.0-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.11.3-py35_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.5*'] None py35_blas_openblas_201
numpy-1.14.1-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.11.2-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.12.1-py36_blas_openblash24bf2e0_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=3.6,<3.7.0a0'] None py36_blas_openblash24bf2e0_201
numpy-1.10.4-py34_blas_openblas_205.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_205
numpy-1.11.1-py34_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.4*'] None py34_blas_openblas_202
numpy-1.8.2-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.14.1-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.9.3-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.10.4-py34_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.4*'] None py34_blas_openblas_203
numpy-1.10.4-py35_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.5*'] None py35_blas_openblas_202
numpy-1.9.3-py35_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_202
numpy-1.13.2-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.11.3-py36_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_202
numpy-1.13.0-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.10.4-py27_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 2.7*'] None py27_blas_openblas_203
numpy-1.13.3-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.1-py35_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 3.5*'] None py35_blas_openblas_202
numpy-1.12.1-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.12.1-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.12.1-py27_blas_openblash24bf2e0_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python >=2.7,<2.8.0a0'] None py27_blas_openblash24bf2e0_201
numpy-1.14.0-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.12.1-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_201
numpy-1.14.1-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.1-py27_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18|0.2.18.*', 'python 2.7*'] None py27_blas_openblas_202
numpy-1.12.0-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.1-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.13.1-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.12.1-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.14.3-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.20,<0.2.21.0a0', 'python >=3.6,<3.7.0a0'] None py36_blas_openblas_200
numpy-1.8.2-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.11.2-py27_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_202
numpy-1.12.0-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.13.0-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.12.1-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.14.2-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.14.0-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.11.1-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.10.4-py34_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_200
numpy-1.13.2-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.13.3-py36_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_200
numpy-1.13.3-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.6*'] None py36_blas_openblas_201
numpy-1.11.1-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.13.3-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.9.3-py36_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.6*'] None py36_blas_openblas_202
numpy-1.11.0-py35_blas_openblas_100.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2*', 'python 3.5*'] None py35_blas_openblas_100
numpy-1.13.3-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.10.4-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.11.3-py27_blas_openblas_202.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_202
numpy-1.11.1-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.9.3-py34_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_203
numpy-1.11.3-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.10.4-py27_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_200
numpy-1.13.1-py34_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.4*'] None py34_blas_openblas_201
numpy-1.11.0-py27_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.18*', 'python 2.7*'] None py27_blas_openblas_201
numpy-1.8.2-py35_blas_openblas_200.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.19|0.2.19.*', 'python 3.5*'] None py35_blas_openblas_200
numpy-1.9.3-py27_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 2.7*'] None py27_blas_openblas_203
numpy-1.9.3-py35_blas_openblas_203.tar.bz2 ['blas 1.1 openblas', 'openblas 0.2.20|0.2.20.*', 'python 3.5*'] None py35_blas_openblas_203
numpy-1.9.3-py36_blas_openblas_201.tar.bz2 ['blas 1.1 openblas', 'openblas >=0.2.15', 'python 3.6*'] None py36_blas_openblas_201
```

**Comment by beckermr** [comment]
Mmmk. I looked at these packages and libgfortran is pulled in from openblas in this case, so we don't have to worry about it. Some of these are old toolchain builds.

**Comment by beckermr** [comment]
This PR is ready for review @conda-forge/core

**Comment by isuruf** [comment]
The way the python script is setup is messy. If there are multiple rules changing `depends` are applied to the same package, only the last will be used.

cc @soapy1

**Comment by beckermr** [comment]
Yes indeed @isuruf. I can refactor, but I am scared of breaking things. Let me do more work on it.

**Comment by beckermr** [comment]
OK @conda-forge/core. I did some work here to clean things up.

Summary of the changes:
 - I changed the way the script works. It now builds an entire new index based on applying the corrections to a deepcopy of the original index. Then the patch instructions are generated at the end of the process by diff'ing the new and original indices. This fixes the issue @isuruf mentioned above where more than one change would not be applied. 
 - I added a test of the function to do the diff'ing of the indices. 
 - I cleaned up the code a bit and added comments.
 
I compared the generated patch files to the latest ones on anaconda.org. The only differences are due to libgfortran changes or new packages that are getting pulled in to the new patches.

I don't think we should do much more work on this code. The new version is more robust and easier to modify. The gains from further testing or cleanup beyond what I did are probably marginal.

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -342,13 +381,14 @@ def _extract_track_feature(record, feature_name):
 
 
 def main():
-
-    base_dir = join(dirname(__file__), CHANNEL_NAME)
+    # not used, leaving because someone might have wanted it?
+    # base_dir = join(dirname(__file__), CHANNEL_NAME)
```
Comment (scopatz) [inline recipe/gen_patch_json.py]: I think it is fine to remove

**Comment by scopatz** [comment]
This looks fine to me, but I would appreciate it if someone else from @conda-forge/core (@jjhelmus?) could take a look too

**Comment by beckermr** [comment]
Hi friends! A quick bump on this @conda-forge/core. I'd love to get this merged so we can start in on rebuilding the stack as needed!

**Comment by scopatz** [comment]
Thanks @beckermr!

**Comment by beckermr** [comment]
Thanks @scopatz! I’m gonna check the numpy deps later today and if things look good then I’ll bug you on Gitter to remove the broken labels.

**Pull request merged**